.github/scripts/install_slices/test_install_slices.py

```diff
@@ -214,14 +216,46 @@ def test_install_slice(self):
         """
         Test install_slice()
         """
-        install_slice("libc6_libs", "amd64", "ubuntu-22.04")
+        mock_missing_copyright = {}
+        install_slice("libc6", "libs", "amd64", "ubuntu-22.04", mock_missing_copyright)
+        assert mock_missing_copyright == {"libc6_libs": False}
         #
         try:
-            install_slice("foo123_bar", "amd64", "ubuntu-22.04")
+            install_slice(
+                "foo123", "bar", "amd64", "ubuntu-22.04", mock_missing_copyright
+            )
             assert False
         except SystemExit as e:
             self.assertEqual(e.code, 1)
 
+    @unittest.mock.patch("os.popen")
+    @unittest.mock.patch("pathlib.Path.is_file")
+    @unittest.mock.patch("apt.debfile.DebPackage.__new__")
+    def test_deb_has_copyright_file(self, mock_debpackage, mock_is_file, mock_popen):
```

Comment:
Thanks for these! Once we land chisel.db we can hopefully simplify these even further.

Reply:
:crossed_fingers: